substrate/client/service/src/builder.rs

```diff
@@ -655,10 +655,13 @@ where
 		(chain, state, child_state)
 	};
 
+	const MAX_TRANSACTION_PER_CONNECTION: usize = 4;
```

Comment:
Nit: the spec only says "at least 4 transactions can be broadcast at the same time". Do we have any reason to also use 4 as the upper bound here?

(One reason I can think of is just that it nudges clients to comply with that bit of the spec, ie if they try to send more than 4, which _might_ not be supported per the spec, then an error will come back)

Reply:
Yep, have changed it to 16